### DISCUSSION TITLE
LAB 06.01 - Task 2

### DISCUSSION THREAD

**Discussion by natandreli** [rramosp/ai4eng.v1#260] [06 - MÉTODOS NO SUPERVISADOS]
Buenas tardes:

La plataforma no me califica bien, a pesar de que mi respuesta es el mismo output esperado.

Mi código es:

```
def mean_sync_move(m):
    from collections import Counter
    result = sum([max(v)/m.shape[0] for v in [(Counter(m[:, c])[-1], Counter(m[:, c])[1]) for c in range(0, m.shape[1])]])/m.shape[1]
    return result
```

Y me sale lo siguiente:

![image](https://github.com/rramosp/ai4eng.v1/assets/110438161/e456e71e-1872-43f9-9380-88e22ddb0b00)

Otro ejemplo:

![image](https://github.com/rramosp/ai4eng.v1/assets/110438161/a38ef331-17ec-49fb-96e5-7f823026325c)

Espero me puedan ayudar.

**Comment by Felipe-RA** [comment]
Saludos @natandreli 

Efectivamente, tu respuesta arroja el resultado correcto.

El error ocurre debido a que tu respuesta es un `float` nativo, pero el corrector espera `np.float64`

Gracias por hacernos saber de este caso, mientras tenemos la oportunidad de agregar este caso de uso al corrector, por favor haz lo siguiente:

```
return np.float64(result)
```

Con esto convertiras el tipo de dato de tu resultado y será calificado correctamente por el corrector.

¡Hasta pronto, hazme saber si tienes algún inconveniente extra!

**Reply by natandreli** [reply]
Gracias :)
